packages/react-components/src/Status/index.tsx

```diff
@@ -144,8 +145,24 @@ function Status ({ className, stqueue = [], txqueue = [] }: Props): React.ReactE
     return null;
   }
 
+  const _onDismiss = (): void => {
+    allst.map(s => s.removeItem());
+    alltx.map(t => t.removeItem());
+  };
+
   return (
     <div className={`ui--Status ${className}`}>
+      {allst.length + alltx.length > 1 && (
```

Comment:
Just be a tiny . bit more explicit here, i.e. prefer the following form -

```js
(allst.length + alltx.length) > 1
```

No functional difference, but when reading the code late at night, it is just immediately obvious
```